go.mod

```diff
@@ -3,53 +3,53 @@ module github.com/kubernetes-csi/external-resizer
 go 1.15
 
 require (
-	github.com/container-storage-interface/spec v1.2.0
+	github.com/container-storage-interface/spec v1.3.0
 	github.com/imdario/mergo v0.3.9 // indirect
```

Comment:
did you intend to update CSI spec or did the update come via k8s client update?

Reply:
@gnufied I deliberately updated it , but forgot to explictly mention it

Reply:
I would rather not update that in this PR.

Reply:
@humblec I see no reason to update CSI spec at last minute within resizer repo. Also have we updated other sidecar's CSI spec version? k8s repo at least is still using 1.3.0.

Reply:
almost all sidears are with CSI spec v1.3.0

Reply:
@gnufied I can revert the change too.. please let me know the choice

Reply:
@gnufied better I reverted..  We will update the SPEC 1.3 accordingly. PTAL